config_src/coupled_driver/ocean_model_MOM.F90

```diff
@@ -213,7 +214,10 @@ module ocean_model_mod
     restart_CSp => NULL()     !< A pointer set to the restart control structure
                               !! that will be used for MOM restart files.
   type(diag_ctrl), pointer :: &
-    diag => NULL()            !< A pointer to the diagnostic regulatory structure
+       diag => NULL()            !< A pointer to the diagnostic regulatory structure
```

Comment:
There are an extra 3 spaces of indenting for diag and diag_IS.  Please follow the MOM6 style guide standards.